src/Sentry.Unity.Native/SentryNative.cs

```diff
@@ -80,7 +83,26 @@ public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentry
             }
         }
         options.CrashedLastRun = () => crashedLastRun;
+
+        // The backend will only be reinstalled if the native SDK has been initialized successfully.
+        // We're handing this off to be called in `BeforeSceneLoad` instead of `SubsystemRegistration` as it's too soon
+        // and the SignalHandler would still get overwritten.
+        ReinstallBackendFunction = SentryNativeBridge.ReinstallBackend;
```

Comment:
This seems hacky: do we change the method in any way? We're just signaling from this part of the case to another. I think a bool is more clear and has other benefits (as in not having the action)